### DISCUSSION TITLE
Ability to switch to dark mode

### DISCUSSION THREAD

**Discussion by ErikMogensen** [Azure/api-management-developer-portal#2575] [Feature request]
## Feature description

It would be great if there was a way to switch to dark mode and then back to light mode. 

## Expected behavior

- A choice somewhere to choose dark or light mode.
- Two modes when setting color in Design mode. 

## Is your portal managed or self-hosted?

At least for the Self-hosted, preferably also for the managed.

**Comment by ghost** [comment]
@ErikMogensen, thank you for opening this issue. We will triage it within the next few business days.

**Comment by ghost** [comment]
@ErikMogensen, thank you for requesting this feature.

**Comment by azaslonov** [comment]
Hi Erik, do I understand correctly, you want your users (of the published portal) to be able to switch to the "dark mode" / another color scheme?

**Comment by ErikMogensen** [comment]
Hi Alex, Yes that is what I'd like.

**Comment by azaslonov** [comment]
Got it, thanks!
